spec/Appendix A -- Persisted Documents.md

```diff
@@ -145,20 +225,62 @@ parameters in one of the manners described in this specification:
 - {extensions} - (_Optional_, map): This entry is reserved for implementors to
   extend the protocol however they see fit.
 
+### Persisted Document Request URL
+
+To enable non-GraphQL HTTP tooling to better integrate with a Persisted Document
+Request, it is recommended that the URL to which a Persisted Document Request is
+sent is a subpath of the _GraphQL endpoint_ containing the {documentId} and the
+{operationName} (if any) separated by a forward slash character (`/`). It is
+recommended that this practice is only followed when {documentId} is a prefixed
+document identifier, since the prefix helps avoid clashes with other subpaths of
+the _GraphQL endpoint_.
+
+Note: By following this practice, traditional HTTP tooling can exercise concerns
+such as caching, rate limiting, audit logging, access-pattern analysis, error
+detection, monitoring and more without needing to fully parse the incoming
+GraphQL request.
+
+For example, if the _GraphQL endpoint_ is `https://example.com/graphql` then a
+persisted document request may be made to an endpoint such as
+`https://example.com/graphql/sha256:517c56d2ba0779653b7698881207f749509f331bdaccbe951a82c378bc869556/FriendNames`.
+For documents containing a single anonymous operation the final segment must be
+omitted, e.g.
+`https://example.com/graphql/sha256:71f7dc5758652baac68e4a10c50be732b741c892ade2883a99358f52b555286b`.
```

Comment:
Should we maybe namespace persisted documents under `/graphq/<prefix>/<sha>`?

e.g.
- `/graphql/persisted/<sha>/<operation_name>`
- `/graphql/documents/<sha>/<operation_name>`
- `/graphql/persisted-documents/<sha>/<operation_name>`

Main reason for this is that `graphql-sse` already encourages `/graphql/stream`, which in theory can conflict with a "custom" hash. Changing this would also leave the options open to add other URL based features later on and not conflict with the persisted documents.

Reply:
We do namespace it in HotChocolate for less collision potential. However, Twitter for instance puts it directly on the GraphQL route as as outlined in this document.

Reply:
I'm not a big fan of having the operation name first. 🤔

From a server perspective, we first look up the hash for the document, and then the operation name within that document. Having the operation name first seems wrong to me.

From a debugging perspective, it does not matter too much whether the operation name comes before the hash.

Reply:
Also if we are looking at Chrome DevTools, the last URL part is displayed. So replacing operation name and hash might even be counter productive.

![image](https://github.com/user-attachments/assets/088f051b-cc81-4896-8213-b44027e5d8ea)

Reply:
Good point, re Chrome devtools. Okay lets stick with the hierarchical approach, it makes more sense anyway :+1: